pkgs/tools/misc/urjtag/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pythonBindings ? false
+, jedecSupport ? false
+, python3 ? null
+, stdenv, autoconf, automake, pkgconfig, gettext, intltool, libtool, bison, flex, fetchgit, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  name = "urjtag";
+  meta = {
+    description = "Enhanced, modern tool for communicating over JTAG with flash chips, CPUs,and many more";
+    homepage = "http://urjtag.org/";
+    license = with stdenv.lib.licenses; [ gpl2Plus lgpl21Plus ];
+    platforms = stdenv.lib.platforms.gnu;  # arbitrary choice
+    maintainers = with stdenv.lib.maintainers; [ lowfatcomputing ];
+  };
+
+  src = fetchgit {
+    url = "git://git.code.sf.net/p/urjtag/git";
+    rev = "7ba12da7845af7601e014a2a107670edc5d6997d";
+    sha256 = "834401d851728c48f1c055d24dc83b6173c701bf352d3a964ec7ff1aff3abf6a";
+  };
+
+  buildInputs = [ gettext pkgconfig autoconf automake libtool makeWrapper ]
+    ++ stdenv.lib.optional pythonBindings python3;
+
+  configureFlags = ''
+    --prefix=/
```

Comment:
> Out of curiosity, did you try flipping "--prefix=/" and "DESTDIR=$out" with "--prefix=$out" and removing DESTDIR?

Just tried that but it doesn't seem to work. So I guess I should go with this method: http://github.com/NixOS/nixpkgs/pull/9533#discussion_r38264557